src/core/hooks/use-form-controller/formts-dispatch.ts

```diff
@@ -18,6 +19,7 @@ export const getInitialState = <Values extends object, Err>({
   const touched = makeUntouchedValues(values);
 
   return {
+    initialValues: Atom.of(values),
```

Comment:
is there a point in making atom out of initial values, since it's not going to be changed?